returnn/tf/layers/signal_processing.py

```diff
@@ -681,10 +737,15 @@ def __init__(self, nr_of_channels=1, **kwargs):
 
     input_placeholder = self.input_data.get_placeholder_as_batch_major()
 
-    output = tf.reshape(input_placeholder, [tf.shape(input_placeholder)[0], tf.shape(input_placeholder)[1], nr_of_channels, tf.shape(input_placeholder)[2] / nr_of_channels])
-    self.output.placeholder = tf.transpose(tf.reshape(tf.transpose(output, [1, 3, 0, 2]), (tf.shape(output)[1], tf.shape(output)[3], tf.shape(output)[0] * tf.shape(output)[2])), [2, 0, 1])
+    output = tf.reshape(input_placeholder,
+                        [tf.shape(input_placeholder)[0], tf.shape(input_placeholder)[1], nr_of_channels,
+                         tf.shape(input_placeholder)[2] / nr_of_channels])
```

Comment:
```suggestion
    output = tf.reshape(
      input_placeholder,
      [tf.shape(input_placeholder)[0], tf.shape(input_placeholder)[1],
       nr_of_channels,
       tf.shape(input_placeholder)[2] // nr_of_channels])
```
Note also the fix for `//` (important!).